### PR TITLE
Tidy up the Shibboleth documentation

### PR DESCRIPTION
**Pull request by settermjd**
- Reflow paragraphs to be one sentence per/line
- Remove unrequired anchors
- Fix up AsciiDoc formatting

### CODE REVIEW

**Comment by mmattel** [comment]
Great stuff. What about if we add a link to https://en.wikipedia.org/wiki/Shibboleth_(Shibboleth_Consortium)
to give an overview what Shibboleth is about?

File: modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc [hunk 1]
```diff
@@ -85,80 +71,55 @@ LoadModule mod_shib /usr/lib64/shibboleth/mod_shib_24.so
...
-and environment mappings by using the `occ` command, like this example
-on Ubuntu Linux:
+After enabling the app it will be in *Not active* mode, which ignores a Shibboleth session and allows you to login as an administrator and inspect the currently available Apache environment variables. 
+Use this mode to set up the environment mapping for the other modes, and in case you locked yourself out of the system. 
+You can also change the app mode and environment mappings by using the `occ` command, like this example on Ubuntu Linux:
 
 ....
 $ sudo -u www-data php occ shibboleth:mode notactive
 $ sudo -u www-data php occ shibboleth:mapping --uid login
 ....
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc]:
```
[source,console]
----
```
removing the $ sign

**Comment by settermjd** [comment]
Changes made, @mmattel.

File: modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc [hunk 2]
```diff
@@ -1,200 +1,141 @@
...
-inspect the currently available Apache environment variables. Use this
-mode to set up the environment mapping for the other modes, and in case
-you locked yourself out of the system. You can also change the app mode
-and environment mappings by using the `occ` command, like this example
-on Ubuntu Linux:
+After enabling the app it will be in *Not active* mode, which ignores a Shibboleth session and allows you to login as an administrator and inspect the currently available Apache environment variables. 
+Use this mode to set up the environment mapping for the other modes, and in case you locked yourself out of the system. 
+You can also change the app mode and environment mappings by using the `occ` command, like this example on Ubuntu Linux:
 
 ....
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc]: [source,console]

File: modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc [hunk 3]
```diff
@@ -1,200 +1,141 @@
...
+
 === Mapping ownCloud User IDs
 
-From 3.1.2 you can now specify a mapper that is used on inbound ownCloud
-user IDs, to adjust them before usage in ownCloud. You can set the
-mapper using `occ`:
+From 3.1.2 you can now specify a mapper that is used on inbound ownCloud user IDs, to adjust them before usage in ownCloud. 
+You can set the mapper using `occ`:
 
 ....
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc]: [source,console]

**Pull request merged**